webview/src/setup/components/studio/Connect.tsx

```diff
@@ -21,26 +18,13 @@ export const Connect: React.FC = () => {
           <a href="https://dvc.org/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token">
             access token
           </a>{' '}
-          generated from your Studio profile page.
+          generated from your Studio profile page. Request a token below or{' '}
+          <button className={styles.buttonAsLink} onClick={saveStudioToken}>
```

Comment:
The style seems different than the previous link (`buttonAsLink` might need an update, underline vs none). Also, it feels weird to have this link here (before the "Get Token" button that is mentioned before), I feel like it should be a button on the same level as the "Get Token" one.

Reply:
[Q] Why do we need this escape hatch? Will the endpoint always return a new token or does it fail if the user has one setup already?

Reply:
> Also, it feels weird to have this link here (before the "Get Token" button that is mentioned before), I feel like it should be a button on the same level as the "Get Token" one.

Makes sense, I can move it to a secondary button. 

> [Q] Why do we need this escape hatch? Will the endpoint always return a new token or does it fail if the user has one setup already?

Yes, the endpoint always creates a new one (your studio profile can hold multiple now) which I why I wanted to leave the option to add an already created one if preferred. If preferred, I can remove the option though if we want to keep  onboarding and simplified as possible.